src/util/contracts.ts

```diff
@@ -9,42 +8,53 @@ import {
 } from "../types/contracts";
 
 // Ethers/Infura
-const infuraUrl = env.get("INFURA_URL").required().asString();
-const provider = new ethers.providers.JsonRpcProvider(infuraUrl);
-const signer = new ethers.VoidSigner(ethers.constants.AddressZero, provider);
+function getVoidSigner(): ethers.VoidSigner {
+  const provider = getEthersProvider();
+  const signer = new ethers.VoidSigner(ethers.constants.AddressZero, provider);
 
-// Contract setup
-const zAuctionAddress = env.get("ZAUCTION_ADDRESS").required().asString();
+  return signer as ethers.VoidSigner;
+}
+
+export function getEthersProvider(): ethers.providers.JsonRpcProvider {
+  const infuraUrl = process.env.INFURA_URL; //.required().asString();
```

Comment:
Can this be changed to env-var? + the other places in this file?

Perhaps this can be refactored to a small helper which can be stubbed if needed `getInfuraUrl()`

Reply:
I changed to use `env-var`, but accessing the env variable `INFURA_URL` only happens here and I don't think that needs to be made into a helper.